src/plugins/discover/public/embeddable/utils/update_search_source.ts

```diff
@@ -37,4 +65,17 @@ export const updateSearchSource = (
   } else {
     searchSource.removeField('fields');
   }
+
+  // if the search source has a parent, update that too based on fetch context
+  const parentSearchSource = searchSource.getParent();
```

Comment:
Why would we need to update a search source parent? When does it happen?

Reply:
When a saved search embeddable is a panel on the dashboard, the search source parent **is the dashboard** - I am updating the parent search source to reflect changes to the dashboard query bar, time picker, controls, etc.